src/components/overview/overview.module.css

```diff
@@ -0,0 +1,34 @@
+.widget {
+  background-color: rgb(235, 150, 150);
+  height: var(--widget-height);
+  background-color: rgb(220, 99, 99);
```

Comment:
Looks like you are setting the background color twice here.

Reply:
Removed extra property